src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -204,5 +204,12 @@
             </property>
             <deprecated>The service "%service_id%" is internal and deprecated since Symfony 3.3 and will be removed in Symfony 4.0</deprecated>
         </service>
+
+        <!-- Commands -->
+        <service id="form.command.debug" class="Symfony\Component\Form\Command\FormDebugCommand">
```

Comment:
we removed the debug requirement from `debug:container` recently, it should be fine to not have it here